app/uk/gov/hmrc/gform/gform/FormController.scala

```diff
@@ -154,7 +157,26 @@ class FormController(
 
             bracket match {
               case Bracket.NonRepeatingPage(singleton, sectionNumber, _) =>
-                validateSections(suppressErrors, sectionNumber)(renderSingleton(singleton, sectionNumber, _))
+                val formModel = formModelOptics.formModelVisibilityOptics.formModel
+                // section numbers with form components that miniSummaryList may refer to
+                val sns = formModel.pageModelLookup
```

Comment:
why do we need this `sns`?

Reply:
this is  to display a `value` of component that is mentioned in `FormCtx` of MiniSummaryListReference. If I don't do that the value is blank.
I tried to fix the blank `value` and found that `summarySection`  validates all sections but I wanted to get only section numbers that needed for MiniSummaryList, thus `sns`